bench/wtperf/wtperf.c

```diff
@@ -243,14 +243,16 @@ op_name(uint8_t *op)
  *     the keys we see are always in order.
  */
 static int
-do_range_reads(WTPERF *wtperf, WT_CURSOR *cursor, int64_t read_range)
+do_range_reads(WTPERF_THREAD *thread, WT_CURSOR *cursor, int64_t read_range)
 {
+    WTPERF *wtperf;
     uint64_t next_val, prev_val;
-    int64_t range;
+    int64_t range, rand_range;
```

Comment:
Alphabetize on the line.

Reply:
@sueloverso, fixed according to your and Don's comments.